core/locals.tf

```diff
@@ -39,14 +39,14 @@ locals {
 
   rds_config = {
     staging = {
-      instance_type            = "db.t3.micro"
+      instance_type            = "db.t3.medium"
```

Comment:
do we really need medium db for staging?

Reply:
Aurora PostgreSQL requires `db.t3.medium` as the smallest instance size. 
https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.SupportAurora

<img width="1198" alt="image" src="https://github.com/sanG-github/nimble-devops-ic-infrastructure/assets/1896814/04a00f84-303e-4e8c-ae67-98f2eee7ae16">

please help checking if the correct value is used in infra-template and correct if possible.

Reply:
got it, thanks

template recommends to use medium instance already: https://github.com/nimblehq/infrastructure-templates/blob/f1612a246b7b8793498d6b9300504a04d3429c7d/templates/addons/aws/modules/rds/variables.tf#L7